dev/conformance/runner.ts

```diff
@@ -177,80 +201,16 @@ const convertInput = {
 /** Converts Firestore Protobuf types in Proto3 JSON format to Protobuf JS. */
 const convertProto = {
   targetChange: type => type || 'NO_CHANGE',
-  commitRequest: commitRequest => {
-    const deepCopy = JSON.parse(JSON.stringify(commitRequest));
-    for (const write of deepCopy.writes) {
-      if (write.update) {
-        write.update.fields = convert.documentFromJson(write.update.fields);
-      }
-      if (write.transform) {
-        write.transform.fieldTransforms = write.transform.fieldTransforms.map(
-            transform => convertProto.transform(transform));
-      }
-      if (write.currentDocument) {
-        write.currentDocument =
-            convertProto.precondition(write.currentDocument);
-      }
-    }
-    return deepCopy;
-  },
-  precondition: precondition => {
-    const deepCopy = JSON.parse(JSON.stringify(precondition));
-    if (deepCopy.updateTime && deepCopy.updateTime.seconds) {
-      deepCopy.updateTime.seconds = Number(deepCopy.updateTime.seconds);
-    }
-    return deepCopy;
-  },
-  transform: transform => {
-    const deepCopy = JSON.parse(JSON.stringify(transform));
-    if (deepCopy.setToServerValue === 'REQUEST_TIME') {
-      deepCopy.setToServerValue = REQUEST_TIME;
-    }
-    return deepCopy;
-  },
-  position: position => {
-    const deepCopy = JSON.parse(JSON.stringify(position));
-    const values: Array<{}> = [];
-    for (const value of position.values) {
-      values.push(convert.valueFromJson(value));
-    }
-    deepCopy.values = values;
-    return deepCopy;
-  },
-  structuredQuery: queryRequest => {
-    const deepCopy = JSON.parse(JSON.stringify(queryRequest));
-    if (deepCopy.where && deepCopy.where.fieldFilter) {
-      deepCopy.where.fieldFilter.value =
-          convert.valueFromJson(deepCopy.where.fieldFilter.value);
-    }
-    if (deepCopy.where && deepCopy.where.compositeFilter) {
-      deepCopy.where.compositeFilter.op = 'AND';
-      for (const filter of deepCopy.where.compositeFilter.filters) {
-        filter.fieldFilter.value =
-            convert.valueFromJson(filter.fieldFilter.value);
-      }
-    }
-    if (deepCopy.startAt) {
-      deepCopy.startAt = convertProto.position(deepCopy.startAt);
-    }
-    if (deepCopy.endAt) {
-      deepCopy.endAt = convertProto.position(deepCopy.endAt);
-    }
-    return deepCopy;
-  },
-  listenRequest: listenRequest => {
+  listenResponse: listenRequest => {
```

Comment:
Note that I kept this because in the default format, running this through the Protobuf library replaced the Enum String constants with their integer equivalent. That caused all sorts of issues.